### ISSUE TITLE
ModuleNotFoundError: "No module named pyTMD.constants"

### ISSUE THREAD

**Issue by edlazarus** [SatelliteShorelines/CoastSeg#241]
Hello, team!

After a long time away, because I was struggling to get the tool to even load (I've abandoned trying to run CoastSeg on a Mac), I'm back to CoastSeg in its much-evolved state. Super exciting to follow along with your progress.

I just reinstalled everything from scratch – instructions, Step 1 – and in Jupyter Lab, upon running the cell to open the Map Dashboard, I got the following:

![pyTMD_error](https://github.com/SatelliteShorelines/CoastSeg/assets/56262802/05a252f4-81c1-425c-8673-3df39ee7a501)

It looks like maybe @2320sharon was in this part of the file tree within the last day – not sure if you're still working in there?

If relevant (but probably not) – I'm working in Windows 10 Pro (v21H2), in Google Chrome (v 121.0.6167.85).

Maybe I missed something silly? Note that _I have not yet_ downloaded the tide model – was trying the basic load first, since instructions list tidal correction as optional.

Cheers!

Eli

**Comment by 2320sharon** [comment]
Hi @edlazarus!
 Its great to have you back trying CoastSeg again!
 
 Thank you for reporting this issue we actually fixed this issue in CoastSeg 1.2.2, but it turns out the conda forge package didn't release properly so this bug was in the version you downloaded. I just finished updating the conda forge package as well as testing CoastSeg 1.2.2 for both our pip and conda packages and this bug should be fixed.
 
 If you used conda to download coastseg ( I'm pretty sure you did ) then use the following command and things should work alright.
```
 conda install -c conda-forge coastseg=1.2.2
```
